roxie/ccd/ccdserver.cpp

```diff
@@ -113,6 +113,47 @@ static const SmartStepExtra dummySmartStepExtra(SSEFreadAhead, NULL);
 
 //=================================================================================
 
+#define NEW_RESTARTABLE_THREAD
```

Comment:
Should this be testing NEW_THREADEDPERSISTENT?

I assume it's in here so that we implement IThreaded and can thus make use of the noew global thread pool code...